packages/aragon-cli/test/deploy/deploy.test.js

```diff
@@ -1,7 +1,9 @@
 import test from 'ava'
 import web3EthAbi from 'web3-eth-abi'
-import { deployContract } from '../../src/lib/deploy'
-import { isAddress, isValidTxHash, getLocalWeb3 } from './test-utils'
+import { expandLink } from '@aragon/toolkit/dist/util'
+//
```

Comment:
Ok, I'll stop reporting these :laughing: 
Maybe do a global search and replace?

Reply:
I saw Daniel did this kind of split between code imported from packages and code imported from internal files. What you think?